### PR TITLE
Request package output change for `flask-security`

### PR DESCRIPTION
**Pull request by nicoddemus**
https://github.com/conda-forge/flask-security-feedstock/pull/7

Perhaps the upload problem is due that the package has changed to `noarch`?

https://github.com/conda-forge/admin-requests/pull/1167 attempted to reset the token, but that did not work.

### CODE REVIEW

File: requests/flask-security.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - flask-security: "flask-security*"
```
Comment (nicoddemus) [inline requests/flask-security.yml:3]: The failure output is:

```
validation results:
{
  "noarch/flask-security-5.5.2-pyhff2d567_0.conda": false
}
```

**Review by beckermr — CHANGES_REQUESTED** [review]
This PR is not needed. We only check the name. Someone from core will need to debug likely.

File: requests/flask-security.yml (hunk 1 above)
Comment (beckermr) [inline requests/flask-security.yml:3]:
```suggestion
  - flask-security: "flask-security"
```

**Review by beckermr — APPROVED** [review]

**Comment by beckermr** [comment]
I checked the feedstock outputs repo and this package is missing. It must have not been registered at some point, but I don't know why. In any case, merging this pr should fix it.

**Comment by nicoddemus** [comment]
Thanks!

**Pull request merged**